akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -1,14 +1,19 @@
 # Classic Cluster Client
 
+@@@ warning
+
+Cluster Client is deprecated in favor of using [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html). 
```

Comment:
Maybe also link to that paragraph: https://doc.akka.io/docs/akka-grpc/current/whygrpc.html#grpc-vs-akka-remoting that explains the trade-offs.